packages/services/api/src/modules/organization/lib/organization-access-token-permissions.ts

```diff
@@ -0,0 +1,67 @@
+import { type PermissionGroup } from './permissions';
+
+export const permissionGroups: Array<PermissionGroup> = [
```

Comment:
This currently only lists the permissions used by the CLI, as we add more functionality to the public, we can add the permissions here. This list is queryable via the GraphQL API, so we can easily show it on the dashboard when creating an organization access token.